perfkitbenchmarker/providers/ibmcloud/ibmcloud_network.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""""Module containing classes related to IBM Cloud Networking.
+
+This module is a place holder, not used currently.  
```

Comment:
This is where network and subnet creation should go. But I believe that you have that all in IBM Cloud API - recommend shifting it here if possible.

Reply:
that's right, I still had that file as a place holder for future but will see if I can move some there.